gen/emit/EmitJulia.cpp

```diff
@@ -236,12 +236,11 @@ struct EmitJulia : public Emitter
     {
         auto* sn = ls.structname.nameUnderscoreCStr();
 
-        emit(0, "global __%s_hash = Int64(0)", sn);
+        emit(0, "const __%s_hash = Ref(Int64(0))", sn);
```

Comment:
Using (and particularly modifying) globals performs poorly in Julia because their type could change at any time, so the compiler can't generate specialized code for them. I replaced the mutable global with a constant Ref. You can think of a Ref like a 0-dimensional array with one element. Rather than mutating the global, we just get and set that single element. In this way, the compiler can be certain that the type of that hash never changes.